recipes/scalene/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set package_name = "scalene" %}
+{% set initial = "s" %}
+{% set version = "1.5.7" %}
+{% set org = "plasma-umass" %}
+{% set author = "emeryberger" %}
```

Comment:
These are used once. Would suggest we inline these

Reply:
Will do.